### DISCUSSION TITLE
how to pass plain text to editor setContent

### DISCUSSION THREAD

**Discussion by stenwolf** [ueberdosis/tiptap#871] [General Questions] [answered]
Hello, I'm trying to use the setContent method in editor. Basically I want to display a message that looks like this: 
`My tag is <xyz>`
So this is what my code looks like
```
const content = 'My tag is <xyz>';
this.editor.setContent(content);
```
And the content is changing to just `My tag is` without the `<xyz>` part.

I have tried to look at the `parseOptions` from ProseMirror, but I dont see anything that says plain text.
How do I just pass plain text to setContent?
Thank you

**Comment by hanspagel** [comment, marked as answer]
setContent expects HTML, and every unknown tag is removed. Try escaping the text before passing it to tiptap.

`My tag is &lt;xyz&gt;` should work. Hope that helps!

**Comment by cp947788039** [comment]
What if I must use tags and do not use escape characters

**Comment by ghost** [comment]
You can use the document as a text
```
Document.extend({
   content: docType == 'text',
})
```

**Comment by Stalinko** [comment]
Another way to insert any content escaped:
```js
editor.setContent({
  type: 'doc',
  content: [
    {
      type: 'paragraph',
      content: [{
        type: 'text',
        text: yourTextHere,
      }],
    },
  ],
})
```
